packages/tooling/cfp-template/template/src/components/welcome/welcome.pw.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { expect, test } from '@playwright/test';
+
+test.describe("fast-welcome", () => {
+    test.beforeEach(async ({ page }) => {
+        await page.goto("/");
+    });
+
+    test("should add a dark class", async ({ page }) => {
+        const element = await page.locator("fast-welcome .dark");
+
+        expect(element).toBeTruthy();
```

Comment:
Is this test running properly? We found that we have to `await expect` or `expect(await `. depending on the context.

Would this work?

```suggestion
        const element = await page.locator("fast-welcome");

        await expect(element).toHaveClass("dark");
```

Reply:
Just realized the selector is finding the child element with `.dark` and not `fast-welcome.dark`. I'll update the suggestion.

Reply:
I tried both, and both work. Is there an issue with doing one or the other?

Reply:
Playwright uses Jest under the hood: 

> `toBeTruthy` matches anything that an if statement treats as true

The `page.locator()` function returns a `Locator` object, and I'm not sure if it's ever considered false.

Reply:
I did your suggestion, I think its more straight forward now that I think about it. I missed the change from toBeTruthy() the first go around I thought we were just looking at the use of await.